tsconfig.json

```diff
@@ -4,7 +4,7 @@
 
     /* Basic Options */
     // "incremental": true,                   /* Enable incremental compilation */
-    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
```

Comment:
Hmmm not sure if this could be meaningful on deployment. Any reasons for changing this now?

Reply:
One of the Set methods wasn't being allowed with ES5. 

I've already merged the front-end for this, so better merge this, but I'll make a note to check this again in develop to make sure it doesn't cause problems. 

Should be okay for back-end though, since we control the execution environment. Would be more risky in front-end since we don't know what browsers people will be using.